README.md

```diff
@@ -80,15 +80,34 @@ bindsym XF86AudioMute exec swayosd-client --output-volume mute-toggle
 # Source volume toggle mute
 bindsym XF86AudioMicMute exec swayosd-client --input-volume mute-toggle
 
+# Volume raise with custom value
+bindsym XF86AudioRaiseVolume + <key> exec swayosd-client --output-volume 15
+# Volume lower with custom value
+bindsym XF86AudioRaiseVolume + <key> exec swayosd-client --output-volume -15
+
+# Sink volume raise with custom value optionally with --device
+bindsym XF86AudioLowerVolume + <key> exec  swayosd-client --output-volume +10 --device alsa_output.pci-0000_11_00.4.analog-stereo.monitor
+# Sink volume lower with custom value optionally with --device
+bindsym XF86AudioLowerVolume + <key> exec  swayosd-client --output-volume -10 --device alsa_output.pci-0000_11_00.4.analog-stereo.monitor
+
+
+
```

Comment:
Remove these extra lines please. Should be 1 empty line between each "category" of binding

Reply:
Spaces removed